.github/workflows/test.yml

```diff
@@ -0,0 +1,48 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+env:
+  FOUNDRY_PROFILE: ci
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    env:
+      NODE_VERSION: 18
+    strategy:
+      fail-fast: true
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with:
+          cache: npm
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
```

Comment:
Does it have to be nightlies? Feels like that's unnecessarily unstable.

Reply:
Sadly there is no 1.0.0 version only nightly for the time being :( https://github.com/foundry-rs/foundry-toolchain